app/views/govuk_wrapper.scala.html

```diff
@@ -0,0 +1,79 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import play.twirl.api.HtmlFormat
+@import views.html.layouts.{main_content, main_content_header, footer}
+@import uk.gov.hmrc.hmrcfrontend.views.html.helpers.HmrcTrackingConsentSnippet
+@import uk.gov.hmrc.hmrcfrontend.config.AssetsConfig
+@import uk.gov.hmrc.hmrcfrontend.views.html.components.HmrcReportTechnicalIssue
+@import uk.gov.hmrc.hmrcfrontend.views.viewmodels.reporttechnicalissue.ReportTechnicalIssue
+
+@this(govUkTemplate: Layout,
+    main_content_header: main_content_header,
+    main_content: main_content,
+    footer: footer,
+    reportAProblemLink: HmrcReportTechnicalIssue,
+    head_content: HmrcTrackingConsentSnippet,
+    assetsConfig: AssetsConfig
+)
+
+@(appConfig: config.AppConfig,
+  title: String,
+  mainClass: Option[String] = None,
+  mainDataAttributes: Option[Html] = None,
+  bodyClasses: Option[String] = None,
+  sidebar: Html = HtmlFormat.empty,
+  contentHeader: Option[Html] = None,
+  mainContent: Html = HtmlFormat.empty,
+  scriptElem: Option[Html] = None)(implicit request: Request[_], messages: Messages)
+
+
+@head = {
+    @head_content()
+    <meta name="format-detection" content="telephone=no" />
+}
+
+@bodyEnd = {
+    @footer(
+      analyticsToken = Some(appConfig.analyticsToken),
+      analyticsHost = appConfig.analyticsHost,
+      ssoUrl = None,
+      scriptElem = None,
+      gaCalls = None)
+}
+
+@mainContentHeader = {
+    @if(contentHeader.isDefined) {
+        @main_content_header(contentHeader = contentHeader.get)
+    }
+}
+
+@getHelpForm = @{reportAProblemLink(ReportTechnicalIssue(serviceId = "silent-login-admin-frontend"))}
```

Comment:
We seem to be ignoring the value set in AppConfig. Again, we still need an appropriate identifier.